test/volume_test.go

```diff
@@ -3,16 +3,19 @@
 package test
 
 import (
+	"fmt"
 	"github.com/hpe-storage/nimble-golang-sdk/pkg/client/v1/nimbleos"
 	"github.com/hpe-storage/nimble-golang-sdk/pkg/param"
 	"github.com/hpe-storage/nimble-golang-sdk/pkg/sdkprovider"
 	"github.com/hpe-storage/nimble-golang-sdk/pkg/service"
-	"github.com/stretchr/testify/assert"
-	"github.com/stretchr/testify/suite"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"	
 	"testing"
 )
 
 const volumeName = "VolumeTest"
+var testResult = "PASS"
+var result ="PASS"
```

Comment:
What is the difference between result and testResult?  More meaningful names might help.  Perhaps this variable could be changed to "testCaseCompleted bool".  Then the somewhat confusing 'result = "FAIL"' statement at the beginning of each test case could be changed to 'testCaseCompleted = false".

Reply:
refactored the code